# goal
string CLASSIFY=classify
string LEARN=learn
string STOP=stop

string request_type #classify, learn or stop
uint32 num_actions
string[] action_names
int32 task_id #add to existing task
---
# result
bool success
---
# feedback
string current_state
string action_name
